sdb/commands/help.py

```diff
@@ -17,24 +17,45 @@
 # pylint: disable=missing-docstring
 
 import argparse
-from typing import Iterable
+from typing import Iterable, Dict, Type
 
 import drgn
 import sdb
 
 
 class Help(sdb.Command):
+    """ Displays help for the specified command for ex: help addr """
 
     names = ["help", "man"]
 
     @classmethod
     def _init_parser(cls, name: str) -> argparse.ArgumentParser:
         parser = super()._init_parser(name)
-        parser.add_argument("cmd", type=str)
+        parser.add_argument("cmd", nargs="?", type=str)
         return parser
 
     def _call(self, objs: Iterable[drgn.Object]) -> None:
-        try:
-            sdb.get_registered_commands()[self.args.cmd].help(self.args.cmd)
-        except KeyError:
-            raise sdb.error.CommandNotFoundError(self.args.cmd)
+        if self.args.cmd is not None:
+            try:
+                cmd = sdb.get_registered_commands()
+                cmd[self.args.cmd].help(self.args.cmd)
```

Comment:
let's rename `cmd` -> `all_cmds`, to match the `else` case.  You could even pull out the `all_cmds = sdb.get_registered_commands()` to happen before the `if`, since it's needed in both cases.

Reply:
thanks!, that's better.